rllib/agents/a3c/a3c_torch_policy.py

```diff
@@ -116,13 +116,6 @@ def loss(
 
         return total_loss
 
-    @override(TorchPolicyV2)
```

Comment:
This is not needed. It does exactly the same thing as the default implementation.